src/lib/paginator/paginator.ts

```diff
@@ -46,6 +46,20 @@ export class PageEvent {
   length: number;
 }
 
+export class PageSizeOption {
```

Comment:
This doesn't have to be a class. You'll get the same strong typing using an interface, but it'll end up generating less ES5 code.